src/collective/cover/interfaces.py

```diff
@@ -30,14 +32,8 @@ def getSearchResults(self, filter_portal_types, rooted, document_base_url, searc
 
 
 class ICoverUIDsProvider(Interface):
-
-    def getUIDs(self):
-        """Get UUIDs associated with the object.
-            could be the UUID of the object or a
-            list of related UUIDs.
-
-        @return: iterable of UUIDs
-        """
+    """GenericUIDsProvider interface will be removed in collective.cover v1.7."""
+    warnings.warn(__doc__, DeprecationWarning)
```

Comment:
No since it's a behavior the user is expecting when adding a container to a list. We can add the adapter to brasil.gov.tiles though, but we think the "deprecation" that was made here is wrong and that's why we're proposing to fix it here.